joomla/joomla-cms
# [contact modal field] Adds Create Contact button and other improvements

Pull Request for Improvement.
### Summary of Changes

This PR is the sequence of https://github.com/joomla/joomla-cms/pull/11830 but for now for contact modal.

For list of changes see https://github.com/joomla/joomla-cms/pull/11830 (same changes, but for contact now).

Similiar PR: 
- Article: https://github.com/joomla/joomla-cms/pull/11830
- Category: https://github.com/joomla/joomla-cms/pull/11857
- Contact: https://github.com/joomla/joomla-cms/pull/11858 (this one)
- Newsfeed: https://github.com/joomla/joomla-cms/pull/11862
### Animated Gif (click to view in full screen)

![modal-contact](https://cloud.githubusercontent.com/assets/9630530/18130250/2cb9e452-6f86-11e6-9e99-7db70862e222.gif)
### Testing Instructions
- Use latest staging
- Apply patch https://github.com/joomla/joomla-cms/pull/11830 and then this patch
- Create a Menu item Single Contact type
- Test all the "Edit", "Create", "Select" and "Clear" buttons  (as animated gif)
- In a multilanguage install edit a contact and try the modal button in the associations tab
- Test all the "Edit", "Create", "Select" and "Clear" buttons.
- Code review to check all is ok.
### Documentation Changes Required

None.
